<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PL0ComplierMainWid</class>
 <widget class="QWidget" name="PL0ComplierMainWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PL0Complier</string>
  </property>
  <widget class="QLineEdit" name="FileUrl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="text">
    <string>输入文件：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchFile">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Compile">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Run">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CodeEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>561</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
